docs/tutorial/part-zero/index.md

```diff
@@ -1,235 +1,201 @@
+Node.js एक ऐसा वातावरण है जो वेब ब्राउज़र के बाहर जावास्क्रिप्ट कोड चला सकता है। Gatsby Node.js के साथ बनाया गया है। Gatsby के साथ उठने और चलने के लिए, आपको अपने कंप्यूटर पर रीसेंट वर्शन इनस्टॉल करने की आवश्यकता होगी।
 
-1. Open your Terminal.
-2. Install Xcode Command line tools by running `xcode-select --install`.
-   - If that fails, download it [directly from Apple's site](https://developer.apple.com/download/more/), after signing-in with an Apple developer account
-3. After being prompted to start the installation, you'll be prompted again to accept a software license for the tools to download.
+_Note: Gatsby's न्यूनतम समर्थित Node.js वर्शन Node 8 है, लेकिन अधिक लेटेस्ट वर्शन का उपयोग करने के लिए स्वतंत्र महसूस करें।_
 
-#### Install Node
+1.  अपना टर्मिनल खोलें।
+1.  Homebrew का लेटेस्ट वर्शन सुनिश्चित करने के लिए `brew update` को चलाएं।
+1.  Node और npm को एक बार में इनस्टॉल करने के लिए इस कमांड को चलाएं: `brew install node`
 
-1. Open your Terminal
-2. Run `brew install node`
-   - If you don't want to install it through Homebrew, download the latest Node.js version from [the official Node.js website](https://nodejs.org/en/), double click on the downloaded file and go through the installation process.
+एक बार जब आपने इंस्टालेशन चरणों का पालन कर लिया, तो सुनिश्चित करें कि सब कुछ ठीक से इनस्टॉल किया गया था:
 
-### Windows Instructions
+### अपने Node.js इंस्टालेशन की जाँच करें
 
-- Download and install the latest Node.js version from [the official Node.js website](https://nodejs.org/en/)
+1. अपना टर्मिनल खोलें।
+2. `node --version` रन करें। (यदि आप कमांड लाइन में नए हैं, तो "कमांड चलाएं" का अर्थ है "कमांड प्रॉम्प्ट में" `node --version` टाइप करें, और एंटर key दबाएं।)
+3. `npm --version` रन करें।
 
-### Linux Instructions
+उन आदेशों में से प्रत्येक का आउटपुट वर्शन संख्या होना चाहिए। आपके वर्शन नीचे दिखाए गए वर्शन के समान नहीं हो सकते हैं! यदि उन आदेशों को दर्ज करना आपको एक वर्शन संख्या नहीं दिखाता है, तो वापस जाएं और सुनिश्चित करें कि आपने Node.js इनस्टॉल किया है।
 
-Install nvm (Node Version Manager) and needed dependencies. nvm is used to manage Node.js and all its associated versions.
+![टर्मिनल में node और npm वर्शन की जांच करें](01-node-npm-versions.png)
 
-_💡 If when installing a package, it asks for confirmation, type `y` and press enter._
 
-#### Ubuntu, Debian, and other `apt` based distros:
+## Git इंस्टॉल करें
 
-1. Run `sudo apt update` and then `sudo apt -y upgrade` to make sure your Linux distribution is ready to go.
-2. Run `sudo apt-get install curl` to install curl which allows you to transfer data and download additional dependencies.
-3. After it finishes installing, run `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.1/install.sh | bash` to download the latest nvm version.
-4. To confirm this has worked, use the following command. `nvm --version`. The output should be a version number.
-5. [Set default Node.js version](#set-default-nodejs-version)
 
-#### Arch, Manjaro and other `pacman` based distros:
```

Comment:
These are new sections added during some synnc with english, can you add translation for these also.

i will suggest revisiting every section one by one so as to find all the new ones